asab/library/providers/git.py

```diff
@@ -172,7 +171,6 @@ def init_task():
 				)
 			else:
 				L.exception("Git repository not initialized: {}".format(err))
-			self.App.stop()
```

Comment:
`return` ... don't claim "library ready" - as discussed.

Reply:
I added returns. But I tested it in the remote control. And remote control works even if the library is not ready. asab.library module does not restrict you to read from the library when library is not ready. So it is up to each microservice to check whether the library is ready before reading. Moreover, in remote control, I subscribe to changes in the Library on library ready. But the Library.ready! signal never comes. So, the remote control is running with wrong set of library layers and without the update of model. Which is serious. 

I think we need to decide whether this should be solved on the asab level or on the microservice level and adjust first. Only then, this can be merged.

```
	async def _on_library_ready(self, event_name, library=None):
		# subscribe to /Site - that's wehere the model files are
		await self.App.LibraryService.subscribe(self.MODEL_PATH)
		self.App.TaskService.schedule(self.update_model_cache())
```